### DISCUSSION TITLE
Import a file that is emited by webpack plugin

### DISCUSSION THREAD

**Discussion by DauMoe** [webpack/webpack#18234] [Q&A]
Hi, Iam totally new in webpack. Trying to create a plugin to create sprite sheet (image + coordinate file). My idea is
- Get all image asset => create a sprite image

- Generate coordinate map and emit or write to disk (.json)

My little plugin works fine but it has a trouble in dev mode

- If I try to write coordinate file to disk (using fs.writeFile, write-file-atom, ...). Webpack devServer will trigger hot reload -> plugin create file again -> devServer is triggered -> ... (infinity looping)

- I try to use `compilation.emitAsset('coordinateMapping.json', <some_data>)`. It can prevent infinity loop but how can I import this file into my code base if it doesn't exists

My plugin can be found [here](https://github.com/DauMoe/svg-sprite-plugin-creator)

**Comment by alexander-akait** [comment]
You should exclude your files from watching, so webpack will not run compilation again
